### PR TITLE
feat(kibbeh): add bot badges to many places

### PR DESCRIPTION
**Pull request by amitojsingh366**
- add bot badges to many places

here's what it looks like:
image
image

### CODE REVIEW

**Comment by BronzW** [comment]
Why can't on the avatar one it just have a bot label next to it's name?

**Comment by nadirabbas** [comment]
much bad

**Comment by amitojsingh366** [comment]
> 
> 
> Why can't on the avatar one it just have a bot label next to it's name?

looks bad

**Comment by amitojsingh366** [comment]
@benawad @ajmnz if you don't like the icons or the placement just tell me on disc, ill change it

**Review by nadirabbas — APPROVED** [review]
much good

File: kibbeh/src/icons/BotIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      viewBox="0 0 512 512"
+    >
+      <path
+        fill="currentColor"
+        d="M256 32C132.8 32 32 132.8 32 256s100.8 224 224 224 224-100.8 224-224S379.2 32 256 32zm-82.16 280A56 56 0 11228 257.84 56 56 0 01173.84 312zm168 0A56 56 0 11396 257.84 56 56 0 01341.84 312z"
+      />
+    </svg>
+  );
+}
+
```
Comment (amitojsingh366) [inline kibbeh/src/icons/BotIcon.tsx:17]: i took this from ionicons btw @ajmnz

**Review by ajmnz — APPROVED** [review]

**Pull request merged**